CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+
+## Version 1.0.0
+* Release date: September 24, 2018
+* Release status: General Availability
+
+## What's new in this version
+* Announcing the SQL Server 2019 Preview extension.
+  * Support for SQL Server 2019 preview features including big data cluster support.
+  * Azure Data Studio Notebooks
+  * The Azure Resource Explorer viewlets you browse data-related endpoints for your Azure accounts and create connections to them in Object Explorer. In this release Azure SQL Databases and servers are supported.
```

Comment:
viewlets  = "view lets"

Reply:
I think that VS Code is using the word "viewlet" though I think this is a word they created (AFAIK).  I don't think they use the term often in user facing documentation through, so probably we should use a better term like Pane or something like that.  Let's discuss when you're back in the office and we can scrub the docs if desired.